code/modules/mob/living/simple_animal/hostile/drakehound.dm

```diff
@@ -0,0 +1,58 @@
+/mob/living/simple_animal/hostile/drakehound_breacher
+	name = "Drakehound Breacher"
+	desc = "A unathi raider with a viscious streak."
+	icon = 'icons/mob/simple_human.dmi'
+	icon_state = "drakehound"
+	icon_living = "drakehound"
+	icon_dead = "drakehound_dead" // Does not actually exist. del_on_death.
+	mob_biotypes = MOB_ORGANIC | MOB_HUMANOID
+	speak_chance = 1
+	turns_per_move = 3
+	death_sound = 'sound/goonstation/voice/deathgasp_2.ogg'
+	response_help = "pushes the"
+	response_disarm = "shoves"
+	response_harm = "slashes"
+	speed = 0
+	maxHealth = 100
+	health = 100
+
+	harm_intent_damage = 5
+	obj_damage = 60
+	melee_damage_lower = 30
+	melee_damage_upper = 30
+	attacktext = "slashes"
+	attack_sound = 'sound/weapons/bladeslice.ogg'
+	minbodytemp = 0
+
+	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
+	speak_emote = list("hisses")
+	loot = list(
+			/obj/effect/decal/cleanable/blood/innards,
+			/obj/effect/decal/cleanable/blood,
+			/obj/effect/gibspawner/generic,
+			/obj/effect/gibspawner/generic)
+	del_on_death = TRUE
+	faction = list("pirate")
+	sentience_type = SENTIENCE_OTHER
+	footstep_type = FOOTSTEP_MOB_SHOE
+
+/mob/living/simple_animal/hostile/drakehound_breacher/Initialize(mapload)
+	. = ..()
+	if(prob(50))
```

Comment:
You probably want ALWAYS spawn blood and vicera, not with 50% chance.

Reply:
The 50% chance is with the pirate salvage. Otherwise, it spawns blood and viscera without.

Reply:
ah, understand